module02/solution.py

```diff
@@ -0,0 +1,70 @@
+# from typing import Any
+
+
+def num_add(a, b):
+    return a + b
+
+def num_sub(a, b):
+    return a - b
+
+def num_mul(a, b):
+    return a * b
+
+def num_div(a, b):
+    return a / b
+
+def num_floor(a, b):
+    return a // b
+
+def num_rem(a, b):
+    return a % b
+
+
+IS_TRUE = True
+
+IS_FALSE = False
+
+PANCAKE_INGREDIENTS = {
+    "flour": 2,
+    "eggs": 4,
+    "milk": 200,
+    "butter": False,
+    "salt": 0.001
+}
+
+def ingredient_exists(ingr, dict):
+    return ingr in dict
+
+def fatten_pancakes(dict):
+    fatten = dict.copy()
+    fatten['eggs'] = 6
+    fatten['butter'] = True
```

Comment:
You could be using the defined above constant here

Reply:
Sure.